### PR TITLE
TECHMD datastream is hard-coded in user messages

### PR DESCRIPTION
**Pull request by robyj**
- Retrieved the techmd_dsid variable and replaced the hardcoded label in messages

Resolves: https://jira.duraspace.org/browse/ISLANDORA-1614

**JIRA Ticket**: (link)

* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)

# What does this Pull Request do?

A brief description of what the intended result of the PR will be and/or what problem it solves.

# What's new?
A in-depth description of the changes made by this PR. Technical details and possible side effects.

Example:
* Changes x feature to such that y
* Added x
* Removed y

# How should this be tested?

A description of what steps someone could take to:
* Reproduce the problem you are fixing (if applicable)
* Test that the Pull Request does what is intended.
* Please be as detailed as possible.
* Good testing instructions help get your PR completed faster.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

Example:
* Does this change require documentation to be updated? 
* Does this change add any new dependencies? 
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? 
* Could this change impact execution of existing code?

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/7-x-1-x-committers

### CODE REVIEW

**Comment by mjordan** [comment]
I'll test this today.

**Comment by DiegoPino** [comment]
@robyj hi, thanks for the pull! Since this is your first pull here, a few checks: do you have a  [CLA](https://github.com/Islandora/islandora/wiki/Contributor-License-Agreements) in place? Pinging @manez too in case its an institutional one. 
Also, could you please fill out the pull request template or remove the items that don't apply. These are pretty useful for future tracking of changes and our release process, even when the change is minor! Thanks again 👍

**Comment by manez** [comment]
Thanks for checking, @DiegoPino. @robyj has an iCLA and coverage from the University of Manitoba's cCLA, so it should be all good 👍

File: includes/derivatives.inc [hunk 1]
```diff
@@ -151,6 +151,7 @@ function islandora_fits_add_datastream($object, $datastream_id, $file_uri) {
   try {
     $ingest = !isset($object[$datastream_id]);
     $mime_detector = new MimeDetect();
+    $techmd_dsid = variable_get('islandora_fits_techmd_dsid', 'TECHMD');
```
Comment (mjordan) [inline includes/derivatives.inc:154]: Should `$techmd_dsid = variable_get('islandora_fits_techmd_dsid', 'TECHMD');` come before the `try`? If it stays here, will it be in scope within the accompanying `catch`?

Comment (mjordan) [reply includes/derivatives.inc:154]: Wrote a little script to test this... it is in scope within the `try`.

**Review by mjordan — APPROVED** [review]
Works as advertised:

**Comment by bryjbrown** [comment]
@mjordan Are we good to merge this then? I can take care of it tomorrow or Monday before branch cutting.

**Comment by mjordan** [comment]
Yes, we're good.

On Oct 20, 2018 7:56 AM, "Bryan J. Brown" <notifications@github.com> wrote:

@mjordan<https://github.com/mjordan> Are we good to merge this then? I can take care of it tomorrow or Monday before branch cutting.

**Pull request merged**